identity-providers.html.md.erb

```diff
@@ -25,16 +25,20 @@ For more information, see [User Account and Authentication LDAP Integration](htt
 
 The [SAML v2 standard](http://saml.xml.org/saml-specifications) is a dominant player in the federated authentication space. It is also one of the harder integrations to configure. When configuring SAML integration on UAA, you must configure both UAA and the SAML IDP. A mistake on either side causes assertions to be rejected and authentication to fail.
 
+## <a id="oidc"></a> OpenID Connect Providers
+
+The [OpenID Connect standard](https://openid.net/developers/specs/)  is a simple identity layer on top of the OAuth 2.0 protocol. It allows Clients to verify the identity of the End-User based on the authentication performed by an Authorization Server, as well as to obtain basic profile information about the End-User in an interoperable and REST-like manner. When configuring OIDC integration on UAA, you must configure both UAA and the OIDC IDP. A mistake on either side causes tokens to be rejected and authentication to fail.
```

Comment:
The term "authorization server" is potentially confusing here. Is this the same thing at the OpenID Connect Provider/IDP? If so, why would an authorization server handle authentication?